chain/vm/syscalls.go

```diff
@@ -286,13 +288,27 @@ func (ss *syscallShim) VerifyAggregateSeals(aggregate proof5.AggregateSealVerify
 	if err != nil {
 		return xerrors.Errorf("failed to verify aggregated PoRep: %w", err)
 	}
+
 	if !ok {
 		return fmt.Errorf("invalid aggregate proof")
 	}
 
 	return nil
 }
 
+func (ss *syscallShim) VerifyReplicaUpdate(update proof7.ReplicaUpdateInfo) error {
+	ok, err := ss.verifier.VerifyReplicaUpdate(update)
```

Comment:
With new ffi imported does this PR already give lotus correct snap deals update proof verification?  Or is there another PR to follow landing this all the way?

Reply:
gives it (hopefully) correct verification!